PSKoans/Koans/Constructs and Patterns/AboutBitwiseOperations.Koans.ps1

```diff
@@ -0,0 +1,546 @@
+        }
+    }
+
+
+    Context 'About the signing bit' {
+        <#
+            A byte is an example of an unsigned integer. The values can range from 0 to 255, the value cannot
+            be a negative number.
+
+            Signed integers reserve a single bit to denote whether the value is positive or negative.
+
+            The signing bit takes the place of the most significant byte.
+
+                    Bit position      Value
+                    ------------      -----
+                               1      Signing bit
+                               2      64
+                               3      32
+                               4      16
+                               5      8
+                               6      4
+                               7      2
+                               8      1
+
+            This value is known as SByte, or signed byte. The signed byte allows numbers in the range -128 to 127.
+
+            In .NET, SByte, Int16, Int32, and Int64 are all signed integers. Byte, UInt16, UInt32, and UInt64 are all
+            unsigned integers.
+        #>
+
+        It 'allows positive values to be expressed in the same way as an unsigned integer' {
+            <#
+                Positive values do not use the signing bit.
+            #>
+
+            '0_______' | ConvertFrom-Binary -To SByte | Should -Be (1 + 4 + 8)
+        }
+
+        It 'uses the most significant bit to describe a negative number' {
+            <#
+                When the signing bit alone is set, the value is the smallest value, -128.
+            #>
+
+            '1000000' | ConvertFrom-Binary -To SByte | Should -Be -128
+
+            <#
+                The opposite binary value is the maximum positive value for the number.
+            #>
+
+            '01111111' | ConvertFrom-Binary -To SByte | Should -Be 127
+
+            <#
+                When the signing bit is set, each value adds to the minimum.
+            #>
+
+            '10000001' | ConvertFrom-Binary -To SByte | Should -Be (-128 + 1)
+            '10000010' | ConvertFrom-Binary -To SByte | Should -Be (-128 + 2)
+            '1_______' | ConvertFrom-Binary -To SByte | Should -Be (-128 + 4)
+            '1_______' | ConvertFrom-Binary -To SByte | Should -Be (-128 + 8)
+        }
+
+        It 'can use -shl to create a negative number from a positive number' {
+            $Value = '00100001' | ConvertFrom-Binary -To SByte
+
+            $Value | Should -Be 33
+            $Value -shl $__ | Should -Be -124
+        }
+    }
+
+    Context 'A matter of perspective' {
+        It 'a binary string can be used to represent different numeric values' {
+            <#
+                The number it represents can depend on the numeric type used.
+            #>
+
+            $BinaryValue = '________'
+
+            $BinaryValue | ConvertFrom-Binary -To Byte | Should -Be 225
+            $BinaryValue | ConvertFrom-Binary -To SByte | Should -Be -31
```

Comment:
Maybe also add one converting the value to int or short, to illustrate how the signing bit's position also depends on the type.